api/src/main/java/com/staxpayments/api/datasource/UserLiveRepository.kt

```diff
@@ -0,0 +1,14 @@
+package com.staxpayments.api.datasource
+
+import com.staxpayments.api.network.NetworkClient
+import com.staxpayments.api.repository.UserRepository
+import com.staxpayments.api.responses.UserResponse
+
+class UserLiveRepository(
+    private val networkClients: NetworkClient
+) : UserRepository {
+
+    override suspend fun getUser(): UserResponse {
+        return networkClients.get("self", responseType = UserResponse.serializer())
+    }
+}
```

Comment:
Always format your code file before you push.. this can cause conflict when trying to merge to master